c/src/detectors/c-string-equality/c-string-equality_non-compliant.c

```diff
@@ -0,0 +1,18 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// {fact rule=c-string-equality@v1.0 defects=1}
+#include <string.h>
+#include <stdio.h>
+
+char *s = "Hello";
+char ss[] = "Hello";
+ //NonCompliant : Using `==` performs character comparison, Which could be leveraged to affect program security.
```

Comment:
In general, please add the comments on the line which is vulnerable, that would help users know what exactly is the problem